web/src/main/kotlin/no/nav/su/se/bakover/web/routes/behandling/BehandlingRoutes.kt

```diff
@@ -238,17 +239,24 @@ internal fun Route.behandlingRoutes(
 
     authorize(Brukerrolle.Attestant) {
 
-        fun kunneIkkeIverksetteMelding(feil: KunneIkkeIverksetteBehandling): Resultat {
-            // funksjon + return: Triks for å få exhaustive when
-            return when (feil) {
+        fun kunneIkkeIverksetteMelding(value: KunneIkkeIverksetteBehandling): Resultat {
+            return when (value) {
                 is KunneIkkeIverksetteBehandling.AttestantOgSaksbehandlerKanIkkeVæreSammePerson -> Forbidden.message("Attestant og saksbehandler kan ikke være samme person")
                 is KunneIkkeIverksetteBehandling.KunneIkkeUtbetale -> InternalServerError.message("Kunne ikke utføre utbetaling")
                 is KunneIkkeIverksetteBehandling.KunneIkkeKontrollsimulere -> InternalServerError.message("Kunne ikke utføre kontrollsimulering")
                 is KunneIkkeIverksetteBehandling.SimuleringHarBlittEndretSidenSaksbehandlerSimulerte -> InternalServerError.message("Oppdaget inkonsistens mellom tidligere utført simulering og kontrollsimulering. Ny simulering må utføres og kontrolleres før iverksetting kan gjennomføres")
                 is KunneIkkeIverksetteBehandling.KunneIkkeJournalføreBrev -> InternalServerError.message("Feil ved journalføring av vedtaksbrev")
-                is KunneIkkeIverksetteBehandling.KunneIkkeDistribuereBrev -> InternalServerError.message("Feil ved bestilling av distribusjon for vedtaksbrev")
                 is KunneIkkeIverksetteBehandling.FantIkkeBehandling -> NotFound.message("Fant ikke behandling")
-                is KunneIkkeIverksetteBehandling.KunneIkkeLukkeOppgave -> InternalServerError.message("Kunne ikke lukke oppgave")
+            }
+        }
+
+        fun iverksattMelding(value: IverksattBehandling): Resultat {
```

Comment:
Hvordan funksjon har type iverksattbehandling dersom vi ikke bruker den til å tilby "noe mer" enn behandlingen i retur?

Reply:
Tanken var å flytte denne logikken ut i web-laget i denne PRen for så å endre frontend/web-laget samtidig senere.

I tillegg synes jeg koden leser bedre og det blir enklere/tydeligere å teste.